doc/source/getting_started.rst

```diff
@@ -76,7 +76,7 @@ First, connect you board with the computer. In the output of :code:`lsusb` you s
 This means the serial adapter is working and there a serial like :code:`/dev/ttyUSB0` should appear.
 
 Next, make sure you have the `Espressif IoT Development Framework <https://github.com/espressif/esp-idf>`_ installed. 
-The current stable (and tested) version is 4.0.
+The current stable (and tested) version is 5.3.
```

Comment:
Maybe don't want to reference the minor version just so it stays up to date longer?

Reply:
Unfortunately we have to use some internal APIs that are not guaranteed to be stable across minor versions, so I'd leave it :+1: